### PR TITLE
Updated README.md

### PR DESCRIPTION
**Pull request by krzyraj**

### CODE REVIEW

**Comment by krzyraj** [comment]
ping @DominikaEZ

**Comment by clash82** [comment]
+1

File: README.md [hunk 1]
```diff
@@ -108,13 +108,13 @@ ShareButtonsBundle currently works with the following social share services:
 
 ## Usage
 
-Place following Twig helper in place where you want to display share buttons bar:
+Insert the following Twig helper in place where you want to display share buttons bar:
```
Comment (DominikaK) [inline README.md]: +Insert the following Twig helper in the place where you want to display the share buttons bar:

**Comment by clash82** [comment]
Thanks @DominikaEZ and @krzyraj :)

**Comment by lserwatka** [comment]
+1

**Pull request merged**